src/components/AddPlaidBankAccount.js

```diff
@@ -253,7 +253,7 @@ class AddPlaidBankAccount extends React.Component {
                                     label: this.props.translate('bankAccount.chooseAnAccount'),
                                 } : {}}
                                 value={this.state.selectedIndex}
-                                hasError={this.getErrors().selectedBank}
+                                errorText={this.getErrorText('selectedBank')}
```

Comment:
I'm not sure if this Picker is working as expected. I still couldn't find where it is located. I think it's inside  http://localhost:8080/bank-account/new (on click `Connect online with Plaid`, is that correct?) but that url throws me a CORS error.

Reply:
Yes, that's the correct path. I linked a thread in my other comment and think you might have to enable the secure staging server AND use the desktop app.